frontend/web/components/EditPermissions.js

```diff
@@ -198,7 +198,7 @@ export default class EditPermissions extends PureComponent {
               <p>
                   Flagsmith lets you manage fine-grained permissions for your projects and environments.
                   {' '}
-                  <ButtonLink href="https://docs.flagsmith.com/permissions/">Learn about User Roles.</ButtonLink>
+                  <ButtonLink href="https://docs.flagsmith.com/advanced-use/permissions" target="new">Learn about User Roles.</ButtonLink>
```

Comment:
What does `target="new"` do? Do you mean `target="_blank"` to open a new tab?

Reply:
Seems like `target="new"` behaves the same way but I think convention is to use `_blank`.